chromium.js

```diff
@@ -4,13 +4,13 @@ const AdmZip = require('adm-zip');
 const execFile = require('child_process').execFile;
 
 function startChromium() {
-  const chromium = '/usr/bin/google-chrome-beta';
+  const chromium = '/usr/bin/google-chrome-stable';
   if (!fs.existsSync(chromium)) {
     console.error('Cannot find chromium');
     return false;
   }
 
-  execFile(chromium, ['--headless', '--disable-gpu', '--remote-debugging-port=9222']);
+  execFile(chromium, ['--headless', '--disable-gpu', '--remote-debugging-address=0.0.0.0', '--remote-debugging-port=9222']);
```

Comment:
`--remote-debugging-address=0.0.0.0` is what I use in my chrome-headless container to handle the situation where Chrome won't bind to the correct container IP/interface (which is less problematic on Linux, but is a problem on Mac). At that point, you'll get connection refused errors in most cases (or just plain crashes).